continuous-integration/linux/collect-and-check-artifacts.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CI_DIR=$(realpath "$SCRIPT_DIR/..")
+ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
+
+# Simple setup
+apt-get update && apt-get install --yes python3 python3-pip || exit $?
+pip3 install twine || exit $?
+
+# Move artifacts into a common directory
+mkdir $ROOT_DIR/dist || exit $?
```

Comment:
Always quote all paths in case there is a whitespace present

Reply:
I'll try to do that where I've added paths. It gets a bit awkward in some places because asterisks (*) are not expanded, and I have to put them outside of the quotes.